docs/configuration/monitoring.md

```diff
@@ -1,77 +1,72 @@
 # Monitoring
 
-Currently two monitoring solutions are supported:
+Currently, we support two monitoring solutions:
 
-* [**Crow**](#setting-up-monitoring-using-crow) - A simple, lightweight, and built-in monitoring solution.
-* [**Prometheus + Grafana + Loki**](#setting-up-monitoring-using-grafana-prometheus-and-loki) - A comprehensive third-party monitoring solution, recommended for more advanced usage.
+-   **[Crow](#setting-up-monitoring-using-crow):** A simple, lightweight, and built-in monitoring solution.
+-   **[Prometheus + Grafana + Loki](#setting-up-monitoring-using-grafana-prometheus-and-loki):** A comprehensive third-party monitoring solution, recommended for more advanced usage.
+
+The sections below provide details on how to set up each monitoring solution.
 
 ## Setting up monitoring using Crow
 
-Crow displays information about the projects pending analysis and the jobs currently running on Codacy.
+Crow displays information about the projects that are pending analysis and the jobs currently running on Codacy.
 
-The Crow tool is installed alongside Codacy after the helm chart is deployed to the cluster.
-It can be accessed through the `/monitoring` path of the url pointing to your Codacy
-installation, e.g. `http://codacy.company.org/monitoring`. You must set the
-`global.codacy.crow.url` value in your `values.yaml` file so that anchor links to your
-projects can be properly established inside `crow`.
+The Crow tool is installed alongside Codacy when the Helm chart is deployed to the cluster and is configured as follows by default:
 
-For example:
+-   Crow is available on the `/monitoring` path of your Codacy installation URL, such as `http://codacy.company.org/monitoring`
+-   The default credentials to access Crow are:
 
-```yaml
-global:
-  codacy:
-    crow:
-      url: "http://codacy.example.com/monitoring"
-```
+    ```yaml
+    username: codacy
+    password: C0dacy123
+    ```
```

Comment:
These default values don't appear on any of the `*.yaml` files provided by us. Is this correct?

Reply:
Yes, it's in the source: https://bitbucket.org/qamine/crow/src/master/.helm/crow/values.yaml#lines-122